playwright-e2e/page-objects/local-authority/laCheckYourAnswer.page.ts

```diff
@@ -0,0 +1,13 @@
+import { Locator, Page } from '@playwright/test';
+
+import BasePage from '../../pages/basePage.page';
+
+export class LACheckYourAnswerPage extends BasePage {
+  readonly checkYourAnswersHeading: Locator;
+
+  constructor(page: Page) {
+    super(page);
+
+    this.checkYourAnswersHeading = page.locator('h1').filter({ hasText: 'Check your answers' });
+  }
```

Comment:
just wondering if a quick way to improve this would be to check for each heading like for the LAChildDetailsPage?  Presumably you could just lift most of the code?

Reply:
That is a good question. There is a part of the heading I want to check as the way the template produces and how Playwright analyses it, more than what is necessary is selected. The h1 locator followed by the filter selects only what I would like to select. Luckily it is not an accessibility fault, though it makes it a bit of a pain for Playwright. I hope that answers your question